src/mongo/MongoTripod.class.php

```diff
@@ -657,11 +633,12 @@ protected function getDataUpdater()
     {
         if(!isset($this->dataUpdater))
         {
+            $readPreference = $this->collection->getReadPreference();
             $opts = array(
                 'defaultContext'=>$this->defaultContext,
                 OP_ASYNC=>$this->async,
                 'stat'=>$this->stat,
-                'readPreference'=>$this->collection->getReadPreference(),
+                'readPreference'=>$readPreference['type'],
```

Comment:
So, the rationale here is that we've only ever allowed setting the type:
https://github.com/talis/tripod-php/blob/master/src/mongo/MongoTripod.class.php#L96
which means the tagsets will always be the defaults when we instantiate MongoTripod, and therefore would be the defaults when we are instantiating MongoTripodUpdates here.

I won't argue that it would be useful to be able to set this, but you've never been able to previously (well, you have, but only be virtue of $tripod->db/$tripod->collection having been public).

It's reasonable to say this is a requirement, otherwise let's create a new issue to figure out how to enable this.